src/Handlers/JavaHandler.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Redberry\Spear\Handlers;
+
+use Redberry\Spear\Interfaces\Data;
+use Redberry\Spear\Interfaces\Handler;
+
+class JavaHandler extends BaseHandler implements Handler
+{
+	public function __invoke(string $code = '', string $input = ''): Data
+	{
+		$this->setImage('openjdk:11');
+		$this->setCode($code);
+		$this->setInput($input);
+		$this->setInterpreter('java');
```

Comment:
Java is compilable language. here should be setCompiler method.
and if there is a proble with file that is produced after compilation you can create setFileToComile method like this setFileToInterpret